Apply ISR to reference particle (optional). (#1167)
BLAST-ImpactX/impactx: examples/incoherent_synchrotron/README.rst

   .. literalinclude:: analysis_bend_isr.py
      :language: python3
      :caption: You can copy this file from ``examples/incoherent_synchrotron/analysis_bend_isr.py``.

.. _examples-bend-isr-ref:

A Single Bend with ISR, Reference Energy Loss

> ==============================================

Reviewer: ```suggestion
A Single Bend with ISR, Reference Energy Loss
=============================================
```

> This is identical to the preceding test, except for the flag ``isr_ref_part = 1``.  In this test, the reference particle experiences radiative energy loss.  For the beam particles,

Reviewer: ```suggestion
This is identical to the preceding test, except for the flag ``isr_ref_part`` option being enabled.  In this test, the reference particle experiences radiative energy loss.  For the beam particles,
```

> This is identical to the preceding test, except for the flag ``isr_ref_part`` option being enabled.  In this test, the reference particle experiences radiative energy loss.  For the beam particles,

Author: ```suggestion
This is identical to the preceding test, except for the flag ``isr_on_ref_part`` option being enabled.  In this test, the reference particle experiences radiative energy loss.  For the beam particles,
```
